### ISSUE TITLE
ATLANTIS_ATLANTIS_URL not correctly picked up from atlantis.fqdn

### ISSUE THREAD

**Issue by MrPiao** [terraform-aws-modules/terraform-aws-atlantis#377]
## Description

I am initializing the terraform-aws-atlantis module with the DNS records for the ALB manually configured (the domain we want is managed outside of Route53). To ensure `local.atlantis_url` is correctly set within the module, I am trying to set `var.atlantis.fqdn` to the correct URL; however, when I run `terraform plan`, the env variable for `ATLANTIS_ATLANTIS_URL` is set to just `https://`.

I have worked around this issue by defining another env variable with `ATLANTIS_ATLANTIS_URL => url we want`, but this is not ideal as AWS squashes the two env variables together, resulting in `terraform plan` always indicating that there is a change to made despite there not being a code change.

Relevant LOC: https://github.com/terraform-aws-modules/terraform-aws-atlantis/blob/f983fd8fb5ada3be696aeb4c4e1329de845a31f9/main.tf#L4

- [x] ✋ I have searched the open/closed issues and my issue is not listed.

## Versions

- Module version [Required]: 4.0.3

- Terraform version: 1.6.3

- Provider version(s): aws 5.28.0

## Reproduction Code [Required]
```
module "atlantis" {
  source  = "terraform-aws-modules/atlantis/aws"
  version = "4.0.3"

  ...

  # ECS Container Definition
  atlantis = {
    fqdn  = "foobar.com"
    image = "ghcr.io/runatlantis/atlantis:v0.26.0"
    ...
  }

  ...

  create_certificate = false
  create_route53_records = false
}
```

## Expected behavior

Env variable `ATLANTIS_ATLANTIS_URL` should be set to "https://foobar.com"

### Plan output (only relevant parts included)
```
      ~ container_definitions    = jsonencode(
          ~ [
              ~ {
                  ~ environment            = [
                      + {
                          + name  = "ATLANTIS_ATLANTIS_URL"
                          + value = "https://"
                        },
                        # (8 unchanged elements hidden)
                    ]
```

**Comment by remiflament** [comment]
Hello,
Did you test this issue with the last version of the module? `4.0.5`

This local env should do the job
```terraform
atlantis_url = "https://${try(coalesce(
    try(var.atlantis.fqdn, null),
    module.alb.route53_records["A"].fqdn,
    module.alb.dns_name,
  ), "")}"
```

On my tests I'm using the creation of the ACM and the record, and it works like a charm. If you need a code example with these, please let me know.

**Comment by bryantbiggs** [comment]
resolved in #379
